arch/arm/core/cortex_m/mpu/nxp_mpu.c

```diff
@@ -491,7 +495,7 @@ static int nxp_mpu_init(struct device *arg)
 	return 0;
 }
 
-#if defined(CONFIG_SYS_LOG)
+#if defined(CONFIG_LOG)
```

Comment:
this is no longer needed. New logger is ready to be used from the start (before any module/driver initialization).